chandra_aca/plot.py

```diff
@@ -304,7 +308,7 @@ def plot_stars(attitude, catalog=None, stars=None, title=None, starcat_time=None
     if catalog is not None:
         _plot_catalog_items(ax, catalog)
     if title is not None:
-        fig.suptitle(title, fontsize='small')
+        ax.set_title(title, fontsize='small')
```

Comment:
I don't recall why this was using a suptitle; does this still look alright for starcheck?